Assets/Scripts/SaveSystem/SaveFileObject.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RogueApeStudio.Crusader.SaveSystem
+{
+    [CreateAssetMenu(menuName = "CustomConfig/SaveFile", fileName = "SaveFile")]
+    public class SaveFileObject : ScriptableObject
+    {
+        [Header("Coins")]
+        public int coins;
```

Comment:
Why aren't thse Private Fields, with public properties?
You can serialize them. This does not comply with the code convention.

Reply:
Apparently this is a unity limitation.